src/styles.ts

```diff
@@ -0,0 +1,5 @@
+export const Z_INDEXS = {
```

Comment:
Can you set a meaningful number instead of "a lot of 9" here?

Reply:
currently, I optimize old code by gathering some z-index into   'Z_INDEXS' variable.   9999... is our old z-index, I have not been changed it yet. 
- Why we need: Z_INDEXS.  it's easy to manage z-index all element of our website.

Reply:
Ok so I think can accept it for now.